windows-iotcore/connect-your-device/Miracast.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
We need to add the new file in the TOC and breadcrumb. Or it will not show up in the side-nav

Reply:
Should be fixed below